helm/values-dev.yaml

```diff
@@ -117,6 +117,7 @@ microservice-chart:
     NPG_URI: "https://localhost/npg"
     NPG_READ_TIMEOUT: "10000"
     NPG_CONNECTION_TIMEOUT: "10000"
+    NPG_PSP_CARDS_PSP_LIST: "" #TODO to be filled with psp id list that are expected to be present into psp api keys configuration
```

Comment:
for now we can configure Intesa in dev/uat env, `BCITITMM`

Reply:
Fixed with https://github.com/pagopa/pagopa-ecommerce-transactions-service/pull/296/commits/2fd09b642a1c360741f5cf8b007f8582bb4b64a2